### PR TITLE
61 enhance the datamodel with a connection between geschäftsjahr and auftrag

### PR DESCRIPTION
**Pull request by Jusdon**

### CODE REVIEW

**Review by timo2o1o — CHANGES_REQUESTED** [review]
Hi @Jusdon,
would you please also develop the second part of the issue? We need a combobox on the management window for orders to be able to make use of this data model changes. Thanks :)!

**Comment by Jusdon** [comment]
> Hi @Jusdon, would you please also develop the second part of the issue? We need a combobox on the management window for orders to be able to make use of this data model changes. Thanks :)!

I add the required commit

File: DoePaAdminViewModel/ManageAuftraegeViewModel.cs [hunk 1]
```diff
@@ -37,6 +37,14 @@ public ObservableCollection<Abrechnungseinheit> Abrechnungseinheiten
             set => SetProperty(ref _abrechnungseinheiten, value, true);
         }
 
+        private ObservableCollection<Geschaeftsjahr> _geschaeftsjahre = new();
+
+        public ObservableCollection<Geschaeftsjahr> Geschaeftsjahre
```
Comment (timo2o1o) [inline DoePaAdminViewModel/ManageAuftraegeViewModel.cs:42]: Please do not bind to this property. Rollback this change and bind the combobox directly to the GeschaeftsjahreViewmodel as we do it in ManageMitarbeiterWindow with the cost centers

File: DoePaAdminApp/Views/ManageAuftraegeWindow.xaml [hunk 2]
```diff
@@ -123,8 +123,11 @@
         </DataGrid>
 
         <StackPanel
+            Orientation="Horizontal"
             Grid.Column="2"
             Grid.Row="3">
+            <Label>Geschäftsjahr:</Label>
+            <ComboBox ItemsSource="{Binding Geschaeftsjahre}" SelectedItem="{Binding SelectedAuftrag.Geschaeftsjahr, Mode=TwoWay}" DisplayMemberPath="Name" />
```
Comment (timo2o1o) [inline DoePaAdminApp/Views/ManageAuftraegeWindow.xaml:130]: Please bind the ItemsSource directly to the GeschaeftsjahreViewmodel.

**Review by timo2o1o — CHANGES_REQUESTED** [review]
Please change the binding of the list with all business years to the GeschaeftsjahreViewModel

File: DoePaAdminApp/Views/MainWindow.xaml [hunk 3]
```diff
@@ -28,7 +28,6 @@
                                           KeyTip="O" />
                     <RibbonApplicationMenuItem Header="Save"
                                           Command="Save"
-                                          ImageSource="/Resources/sd_card_32x32.png"
```
Comment (timo2o1o) [inline DoePaAdminApp/Views/MainWindow.xaml:31]: Totally forgot about that one :D

File: DoePaAdminApp/Views/ManageAuftraegeWindow.xaml [hunk 4]
```diff
@@ -103,6 +103,7 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellEditingTemplate>
                 </DataGridTemplateColumn>
+                <DataGridComboBoxColumn Header="Geschäftsjahr" ItemsSource="{Binding Source={StaticResource Locator}, Path=ManageGeschaeftsjahreViewModel.Geschaeftsjahre, Mode=TwoWay}" SelectedItemBinding="{Binding Path=ZugehoerigesGeschaeftsjahr}" DisplayMemberPath="Name"/>
```
Comment (timo2o1o) [inline DoePaAdminApp/Views/ManageAuftraegeWindow.xaml:106]: This is what I had in mind :D.

**Review by timo2o1o — APPROVED** [review]
Made some adjustments in another commit.

**Comment by Jusdon** [comment]
> Made some adjustments in another commit.

Are you make these changes in another commit ? Or I ?

**Comment by timo2o1o** [comment]
> > Made some adjustments in another commit.
> 
> Are you make these changes in another commit ? Or I ?

I did, in this commit:
https://github.com/doerffler/DoePaAdmin/pull/63/commits/c2a4d9908cf986cbe6e4581172de41c60f02e461

**Pull request merged**